src/shared/ui/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,24 @@
+import react, { useState } from "react";
+import './GenerSelect.css';
+
+const geners = ['Action', 'Comedy', 'Drama', 'Fantasy'];
+
+export const GenreSelect = ({ genres = [], onSelect }) => {
+
+  const handleButtonClick=(event)=>{
+    onSelect(event.target.value);
+  }
+
+  return (
+    <nav>
+      <ul>
+        {genres?.map((val,i) =>{
```

Comment:
it's better to use a human readable names for items 
```suggestion
        {genres?.map((genre, index) =>{
```

Reply:
fixed